src/lib/MVC/Symfony/Security/InteractiveLoginToken.php

```diff
@@ -14,32 +15,73 @@
  */
 class InteractiveLoginToken extends UsernamePasswordToken
 {
-    /** @var string */
-    private $originalTokenType;
+    private ?TokenInterface $originalToken = null;
+
+    private string $originalTokenType;
 
     public function __construct(UserInterface $user, $originalTokenType, $credentials, $providerKey, array $roles = [])
     {
         parent::__construct($user, $credentials, $providerKey, $roles);
+
         $this->originalTokenType = $originalTokenType;
     }
 
-    /**
-     * @return string
-     */
-    public function getOriginalTokenType()
+    public function getOriginalTokenType(): string
     {
         return $this->originalTokenType;
     }
 
+    /**
+     * @return array{
+     *     0: string,
+     *     1: mixed,
+     *     2: null|\Symfony\Component\Security\Core\Authentication\Token\TokenInterface
```

Comment:
You can omit numeric indices, if all are supplied (even when null). In contrast to this you would need to declare index that is "fully nullable" (not passed at all) and add `?` to the declaration.

For example `2?: null|TokenInterface`.

```suggestion
     *     string,
     *     mixed,
     *     null|\Symfony\Component\Security\Core\Authentication\Token\TokenInterface
```